### DISCUSSION TITLE
Reporting hate speech

### DISCUSSION THREAD

**Discussion by MrUpsidown** [community/community#122586] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Earlier this year, I stumbled upon a page hosted on github.io that kind of shocked me. In essence, the page shows a huge Israel flag and a short text that clearly takes sides in the ongoing conflict between Israel and Gaza with very harsh words.

I reported it as hate speech on February 15, and again on April 29 but no action was ever taken. Is there anything else I should do?

**Comment by MrUpsidown** [comment]
Right. I thought Github was not the place to publicly share political opinions, propaganda and hate speech but I guess I was wrong.

**Comment by InWamos** [comment]


**Reply by MrUpsidown** [reply]
I reported it twice, on February 15. and April 29. via the official reporting channel.

* Thank you for contacting GitHub Support. We wanted to let you know that we've received your message. Ticket ID: 2599057
* Thank you for contacting GitHub Support. We wanted to let you know that we've received your message. Ticket ID: 2754091

**Comment by MrUpsidown** [comment]
Can I share the page link here so that other users can report it?

**Reply by MrUpsidown** [reply]
I'll take the few upvotes for a _yes_. Here is the page : https://ronenness.github.io/3D-Box-Shot-Maker/

Please feel free to report it if you also think this has nothing to do on GitHub.

**Comment by adnanbaqi** [comment]
When dealing with content hosted on platforms like GitHub Pages, here are a few steps you can consider taking if your previous reports haven't resulted in the desired outcome:

- Follow Up on Your Report: Sometimes, following up on your initial report can help. You can contact GitHub support directly to inquire about the status of your reports. Provide them with as much information as possible, such as the URL of the page, the dates you submitted your reports, and any other relevant details. This might prompt a reevaluation of the content.

- Provide Detailed Context: When reporting content, be as detailed as possible about why you believe the content should be considered hate speech. Specific excerpts or descriptions of the content and an explanation of how it violates GitHub's policies can strengthen your report.

- Review GitHub's Policies: Ensure that the content you are reporting actually violates GitHub's specific guidelines. GitHub has a detailed policy regarding what constitutes hate speech and harassment. Familiarizing yourself with these guidelines can help you make a stronger case in your report.

- Reach Out Through Social Media: Sometimes raising awareness about the issue on social media platforms can lead to quicker action, especially if the content clearly violates community guidelines. Public pressure can sometimes prompt a reevaluation by the hosting service.

- Legal Advice: If the content is particularly egregious and could be violating local laws, you might consider seeking legal advice. Legal professionals can offer guidance on whether there are any legal actions you can take based on your jurisdiction.

- Engage with Community Advocacy Groups: If the content is offensive or harmful to a particular group, reaching out to advocacy groups can help. They might have more experience and resources to deal with the issue effectively.
